aiosparql/client.py

```diff
@@ -166,3 +169,20 @@ def _pretty_print_query(self, query: str) -> str:
                     code=exc.code, message=exc.message, headers=exc.headers,
                     explanation=explanation)
             return await resp.json()
+
+    @property
+    def closed(self):
+        return self._closed
```

Comment:
For some reason also linked to `_CoroGuard`, I cannot simply return `self.session.closed`